### PR TITLE
fix configuration to match ipv6 address

### PR DESCRIPTION
**Pull request by caiqi1111**
**Please provide a description of this PR:**

### CODE REVIEW

**Comment by zirain** [comment]
cc @zhlsunshine would you please help to confirm this?

File: samples/addons/prometheus.yaml [hunk 1]
```diff
@@ -127,6 +127,13 @@ data:
         - __address__
         - __meta_kubernetes_service_annotation_prometheus_io_port
         target_label: __address__
+      - action: replace
```
Comment (howardjohn) [inline samples/addons/prometheus.yaml:130]: This file is generated from Prometheus helm chart (manifests/addons/values-prometheus.yaml). Can we fix it upstream? Or if it is, just update the version we use

Comment (caiqi1111) [reply samples/addons/prometheus.yaml:130]: Prometheus Community has update the configuration to match ipv6 (https://github.com/prometheus-community/helm-charts/pull/2127/files). Maybe this file should have similar update?

Comment (howardjohn) [reply samples/addons/prometheus.yaml]: The only accepted changes are in manifests/addons/gen.sh or manifests/addons/values-prometheus.yaml. The file being modified is generated from those

Comment (caiqi1111) [reply samples/addons/prometheus.yaml]: Thanks. I understand how to change this file now. Prometheus helm-charts version should be 15.9.3 or higher to support IPV6.

**Pull request closed without merging**